src/pages/_app.tsx

```diff
@@ -7,28 +7,38 @@ import { googleAnalyticsScript, gtag } from '@/constant';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import { SessionProvider } from 'next-auth/react';
 import { SpeedInsights } from '@vercel/speed-insights/next';
+import useUser from '@/hooks/useUser';
+import { useOnboardingRedirect } from '@/hooks/useOnboardingRedirect';
```

Comment:
we always do relative importing from ROOT. sop '@/hooks/useOnboardingRedirect' -> '@/hooks';